src/components/App.tsx

```diff
@@ -6005,6 +6012,7 @@ class App extends React.Component<AppProps, AppState> {
             maybeGroupAction && actionGroup,
             mayBeAllowUnbinding && actionUnbindText,
             mayBeAllowBinding && actionBindText,
+            mayBeAllowToggleLineEditing && actionToggleLineEditor,
```

Comment:
we should put the item into the group with lock/duplicate as that's functionally closer than adding to a library

![image](https://user-images.githubusercontent.com/5153846/192304522-adbea0b5-ebda-4eb1-a786-f5cca59bb948.png)